django/db/models/functions/base.py

```diff
@@ -205,3 +205,18 @@ def as_oracle(self, compiler, connection):
 class Upper(Transform):
     function = 'UPPER'
     lookup_name = 'upper'
+
+
+class Ltrim(Transform):
```

Comment:
Instead of placing them here, perhaps creating a `text.py` file? The current documentation could also be organized a bit better to have these text related functions under a single section.

Reply:
Agreed. But it should be done in advance in a separate PR.